### PR TITLE
Clean up VGUI speedometer content

### PR DESCRIPTION
**Pull request by geight**
Closes https://github.com/momentum-mod/docs/issues/239.

### CODE REVIEW

File: _posts/guides/2020-05-29-guide_override-custom-assets.md [hunk 1]
```diff
@@ -53,21 +53,6 @@ For example, renaming `weapon_momentum_stickylauncher_tf2.txt` to `weapon_moment
...
-The resource file for the speedometer is `speedometer.res`, located in `momentum/resource/ui/`.
-Copy this file to `momentum/custom/overrides/resource/ui/` to override the game's base file.
-From this you can modify each speedometer's position, font, width, background color, etc.
-
-For example, changing the `"ypos"` field under `"HudSpeedMeter"` changes the vertical position of all speedometers.
-To completely center hud elements, they will have to be moved upwards by half their height. 
-A good value to center the speedometer with the default font is `"c-8"`, which sets the speedometer 8 pixels (half the height of the default font text) above the center.
-
 ## Reloading Within the Game
 To reload all control resource files, including the ones you are overriding, use the command [`{{ page.ccom_resreload }}`](/command/{{ page.ccom_resreload }}).
```
Comment (braem) [inline _posts/guides/2020-05-29-guide_override-custom-assets.md:57]: can remove this and the `ccmon_resreload` metadata as that is VGUI remnants

File: _posts/guides/2020-09-05-guide_gamemode-specific-cfgs.md [hunk 2]
```diff
@@ -7,86 +7,19 @@ category: guide
...
-  ...
-}
-```
-
-As a start, feel free to copy this example, or simply copy the entire contents of `momentum/resource/ui/Speedometer.res` into the `"layout"` field. 
+Config files themselves are a list of lines to be entered into the console and are usually used to set console variables and binds. A collection of console variables can be found [here](/categories/#var).
 
-The layout can be reloaded within the game by using [`{{ page.ccom_speedoload }}`](/command/{{ page.ccom_speedoload }}). 
-Alternatively, [`{{ page.ccom_resreload }}`](/command/{{ page.ccom_resreload }}) will reload all control resource files, including the custom, gamemode-specific speedometer layout.
+The layout can be reloaded within the game by using [`{{ page.ccom_resreload }}`](/command/{{ page.ccom_resreload }}) which will reload all control resource files.
```
Comment (braem) [inline _posts/guides/2020-09-05-guide_gamemode-specific-cfgs.md:25]: Same here

**Review by braem — CHANGES_REQUESTED** [review]

File: _posts/guides/2020-05-29-guide_override-custom-assets.md (hunk 1 above)
Comment (geight) [reply _posts/guides/2020-05-29-guide_override-custom-assets.md]: Addressed in [`52cf91e` (#245)](https://github.com/momentum-mod/docs/pull/245/commits/52cf91e9750e40f68ac371e8ca0d98adc89efb23)

File: _posts/guides/2020-09-05-guide_gamemode-specific-cfgs.md (hunk 2 above)
Comment (geight) [reply _posts/guides/2020-09-05-guide_gamemode-specific-cfgs.md]: Addressed in [`52cf91e` (#245)](https://github.com/momentum-mod/docs/pull/245/commits/52cf91e9750e40f68ac371e8ca0d98adc89efb23)

**Review by braem — APPROVED** [review]

**Pull request merged**